src/components/Item/Item.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component, ReactNode } from 'react';
+import { ResponseItem } from '../../types';
+import classes from './Item.module.scss';
+import platformsSlugData from '../../utils/platformsSlugData';
+
+export default class Item extends Component<{ item: ResponseItem }> {
+  changeClassName = (slug: string) => {
+    const platformsSlug = platformsSlugData;
+    const currentClassName = platformsSlug[slug] || 'platform_template';
+    return `${classes.platform_logo} ${currentClassName}`;
+  };
+
+  render(): ReactNode {
+    const { item } = this.props;
+    return (
+      <div className={classes.item}>
+        <div className={classes.item_image_wrapper}>
+          <img
+            className={classes.item_image}
+            src={item.background_image || '/fallback.png'}
+            alt={`${item.name}_image`}
+          />
+        </div>
+        <div className={classes.item_title}>
+          <h3 className={classes.item_heading}>{item.name}</h3>
+          {item.metacritic ? (
+            <p className={classes.item_rate}>
+              Metacritic: <strong>{item.metacritic} / 100 </strong>
+            </p>
+          ) : (
+            ''
+          )}
```

Comment:
Can it be changed like this?
![image](https://github.com/getgitgood/react-rss/assets/55094494/02538bce-861a-43f9-9477-4584c37f6155)

Reply:
that's better, no shit. much appreciated. fixed